### ISSUE TITLE
[bug] import hook ignores functions with only keyword-only args

### ISSUE THREAD

**Issue by avolchek** [beartype/beartype#345]
Hi!
It seems that Beartype import hook ignores (doesn't generate type-checking wrapper) functions with only keyword-only arguments and without return type annotation. It can be reproduced with the following snippet:

```python
# module.py

def foo_bad(*, arg: str):
    print(arg)

def foo_good(arg: str):
    print(arg)

-----------------------------------------------

# main.py

from beartype.claw import beartype_all

beartype_all()


from module import foo_bad, foo_good

# passes type-checking
foo_bad(arg=42)

# beartype.roar.BeartypeCallHintParamViolation: Function module.foo_good() parameter arg=42 violates type hint <class 'str'>, as int 42 not instance of str.
foo_good(arg=42)



```

**Comment by leycec** [comment]
**Oh, Gods!** You're so right. This is appalling. It's been years since I've fielded an issue like this. I thought we'd finally nailed the low-level fundamentals of @beartype down to the rotting floorboards of GitHub. I was wrong. Now, we all pay the terrifying price.

Thanks for the superb minimal reproducible example (MRE), @avolchek. I'll resolve this immediately. If a miracle happens, I'll push this out with our new upcoming `beartype 0.18.0` release. It may be cold, but I'm still sweating through my flannel. :cold_sweat:

**Comment by leycec** [comment]
**Resolved by 3926b09acc4c.** Praise Hypnobear! All is well. This has been resolved for our upcoming @beartype 0.18.0 release – to be published before I lose my last hair follicle. So... *any day now.*

Until then, please circumvent this shameful display of personal ineptitude and moral failing by **annotating returns** (i.e., typing the return values of *all* callables). Of course, annotating returns is really something everybody wants to do anyway. Now, you really *really* want to:

```python
# Instead of this unannotated return...
def foo_bad(*, arg: str):  # <-- wat? no "-> None"!? O_O
    print(arg)

# Please annotate this return! Voilà. Experience the magic of worky.
def foo_bad(*, arg: str) -> None:  # <-- all's right with the world
    print(arg)
```

I apologize for this inconvenience with a scrunched-up face that's inducing a horrible migraine.

![](https://media1.tenor.com/m/B3GXjLzHbh4AAAAC/the-office-michael-scott.gif)
